src/Schemes/AffineSchemes/Objects/Attributes.jl

```diff
@@ -672,7 +670,7 @@ function _singular_locus_with_decomposition(X::AbsSpec{<:Field, <:MPAnyQuoRing},
     minvec = minors(M, n-d)
     J = ideal(R, minvec)
     JX = ideal(OO(X),minvec)
-    one(OO(X)) in JX && return result
+    one(OO(X)) in JX && return empty
```

Comment:
This is the line that usually kills everything: If the ideal of minors is just too big, we have no chance to go through with this. But there's no way avoiding it, is there? I mean: The Jacobi-criterion is just what it is, right?

Reply:
Yes, indeed. this is a Groebner Basis computation which can easily get out of hand.
We can get around the Jacobian criterion completely in the application K3.jl, if we use a Hironaka-style invariant for resolution -- but for the time being (until the other line of development has reached this), there is no way around the Jacobian criterion.